app/models/dispatcher.rb

```diff
@@ -99,10 +96,16 @@ def on_comment_created(comment)
     end
   end
 
-  def on_proposal_update(_proposal)
+  def on_proposal_update(modifier)
+    if proposal.client_slug == "ncr"
```

Comment:
if I fat-finger while coding and type this:

```
if proposal.client_slug == "mcr"
```

and then wonder why my tests are failing (or worse! not failing), that's more brittle than mistyping `proposal.mcr?` which will raise a method not found error (catastrophic failure being a Good Thing).

i.e. if we're going to hard-code client-specific things into client-agnostic classes, why not make the client-specific code into a method that is harder to mistype.